From 9eaa7c820e4dc6b90fa91f6d9f1b32d83cece391 Mon Sep 17 00:00:00 2001
From: TheKit <nekit1000@gmail.com>
Date: Tue, 25 May 2021 21:28:24 +0200
Subject: [PATCH 07/12] (halium) add android.media.ICameraRecordService to
 manual interface whitelist

Needed for Ubuntu Touch camera record service

Change-Id: I9621e97bf209dd15d11d046553c1ab0bd6d16480
---
 libs/binder/include/binder/IInterface.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libs/binder/include/binder/IInterface.h b/libs/binder/include/binder/IInterface.h
index ac845bc..a5ed791 100644
--- a/libs/binder/include/binder/IInterface.h
+++ b/libs/binder/include/binder/IInterface.h
@@ -242,6 +242,7 @@ constexpr const char* const kManualInterfaces[] = {
         "android.hardware.IStreamListener",
         "android.hardware.IStreamSource",
         "android.media.IAudioService",
+        "android.media.ICameraRecordService",
         "android.media.IDataSource",
         "android.media.IDrmClient",
         "android.media.IMediaCodecList",
-- 
2.45.2

